kiwi/repository/yum.py

```diff
@@ -248,9 +258,9 @@ def _create_runtime_config_parser(self):
         self.runtime_yum_config.set(
             'main', 'gpgcheck', self.gpg_check
         )
-        self.runtime_yum_config.set(
-            'main', 'repo_gpgcheck', self.gpg_check
-        )
+        # We are not setting repo_gpgcheck, which forces repository
+        # signature checks, because I could not manage to make it work.
+        # Nevertheless, package signature validation is functional.
```

Comment:
Just tested with CentOS 7 JeOS with the same result. Using:

`yum-3.4.3-150.el7.centos.noarch`